doc/nrf/known_issues.rst

```diff
@@ -222,7 +223,7 @@ nRF5
 nRF5340
 =======
 
-.. rst-class:: v1-7-1 v1-7-0
+.. rst-class:: v1-8-0 v1-7-1 v1-7-0
```

Comment:
@de-nordic this is fixed with the latest upmerge right? - with https://github.com/apache/mynewt-mcumgr/pull/141

Reply:
No changes made.